doc/user/content/release-notes.md

```diff
@@ -61,6 +61,23 @@ Wrap your release notes at the 80 character mark.
   [`WITH` option](https://materialize.io/docs/sql/create-source/avro-kafka/#with-options)
   from sources. Materialize now automatically selects the optimal batch size.
   **Backwards-incompatible change.**
+  
+<span id="v0.4.2"></span>
+## v0.4.2 (Unreleased)
+
+- Output [SHOW COLUMNS](/docs/sql/show-columns/) in the same order as the source {{% gh 4084 %}}.
+
+- Allow enable/disable of memory profiling without restart {{% gh 4005 %}}.
+
+- Reduce memory usage of outer equijoins {{% gh 4047 %}}.
+
+- Fix panic on [INSERT INTO with JSONB](/docs/sql/insert/) {{% gh 4097 %}}.
+
+- Fix support for large VALUES queries {{% gh 3995 %}}.
```

Comment:
nit: s/VALUES/`VALUES`/

Reply:
this just meant wrapping the word VALUES in backticks to apply code formatting

Reply:
thanks for the callout!  i wasn't sure if there was a better doc page i should link to as well